### PR TITLE
feat: adding tokenization

### LINKED ISSUE #6: CreditCard tokenization
Implement the tokenization flow on store checkout page

### PR DESCRIPTION
**Pull request by ghost**
Todo: Change sdk URL to CDN

closes #6

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -171,24 +209,29 @@
...
             infinitePayFormHandler
         )
+
+        function init() {
+            if(!!window["IPay"]) return;
+
+            var head = document.getElementsByTagName("head")[0];
+            var script = document.createElement("script");
+            script.async = 1;
+            script.src = "https://ipayjs.infinitepay.io/production/ipay-latest.min.js";
```
Comment (panachi-cw) [inline src/index.js:223]: The src will be different for production and sandbox, right?

File: includes/class-wc-wooinfinitepay-init.php [hunk 2]
```diff
@@ -223,7 +233,7 @@ public function payment_scripts() {
 			'woocommerce_infinitepay',
 			'wc_infinitepay_params',
 			array(
-				'uuid' => vsprintf( '%s%s-%s-%s-%s-%s%s%s', str_split( bin2hex( random_bytes( 16 ) ), 4 ) ),
+				'token' => $this->sandbox === 'yes' ? $this->sandbox_api_key : $this->api_key
```
Comment (panachi-cw) [inline includes/class-wc-wooinfinitepay-init.php:236]: Can we use `api_key` for now?

**Review by panachi-cw — CHANGES_REQUESTED** [review]

File: includes/class-wc-wooinfinitepay-init.php (hunk 2 above)
Comment (joaoescudero) [reply includes/class-wc-wooinfinitepay-init.php]: Done 👍

File: src/index.js (hunk 1 above)
Comment (joaoescudero) [reply src/index.js]: Done 👍

**Review by panachi-cw — APPROVED** [review]

**Pull request merged**